unified-scheduler-pool/src/lib.rs

```diff
@@ -1212,7 +1209,7 @@ impl<S: SpawnableScheduler<TH>, TH: TaskHandler> ThreadManager<S, TH> {
         self.scheduler_thread = Some(
             thread::Builder::new()
                 .name("solScheduler".to_owned())
-                .spawn_tracked(scheduler_main_loop())
```

Comment:
this is similar to this: https://github.com/anza-xyz/agave/pull/1211#discussion_r1600332581